receiver/postgresqlreceiver/testdata/config.yaml

```
@@ -0,0 +1,19 @@
+receivers:
+  postgresql:
+    collection_interval: 10s
+    username: otel
+    password: otel
+    database: otel
+    endpoint: 127.0.0.1:5432
+
+processors:
+
+exporters:
+    loglevel: debug
```

This does not appear to be valid as is.
```suggestion
exporters:
  logging:
    loglevel: debug
```

---

fixed